crates/egui_demo_lib/src/demo/table_demo.rs

```diff
@@ -109,6 +109,7 @@ impl super::View for TableDemo {
         StripBuilder::new(ui)
             .size(Size::remainder().at_least(100.0)) // for the table
             .size(Size::exact(10.5)) // for the source code link
```

Comment:
alternatively, we change this to reserve `TextStyle::Body.resolve(ui).height()` instead of the hard-coded 10.5

Reply:
Done